contracts/bridge/BridgeTransferERC721.sol

```diff
@@ -64,7 +66,7 @@ contract BridgeTransferERC721 is BridgeTokens, IERC721BridgeReceiver, BridgeTran
         );
 
         if (modeMintBurn) {
-            require(ERC721MetadataMintable(_tokenAddress).mintWithTokenURI(_to, _tokenId, _tokenURI), "mint failed");
+            IERC721Mint(_tokenAddress).mintWithTokenURI(_to, _tokenId, _tokenURI);
```

Comment:
OMG. I'm sorry I forgot to upload it. `IERC721Mint` is an interface and `mintWithTokenURI` is a function. The mint behavior is defined by each token contract. Please refer to description no5 for the detail. (FYI, the previous tests passed are because of the precompiled bindings)